expression/expression.go

```diff
@@ -36,6 +36,14 @@ const (
 	scalarFunctionFlag byte = 3
 )
 
+// Flag represents expression's property.
+type Flag byte
+
+const (
+	// FlagHoldChunkMemory indicates expression return result maybe returned by unsafe operation over chunk.
+	FlagHoldChunkMemory Flag = 1 << iota
```

Comment:
By default, this is true. I think we can remove this flag and the `Flag()` API from the `expression` interface.